### ISSUE TITLE
Small bug in treasure hunt's vertex shader

### ISSUE THREAD

**Issue by khrismuc** [googlevr/gvr-android-sdk#268]
The color is calculated using

```
v_Color = a_Color * diffuse;
```

which also changes the opacity. Should be:

```
v_Color = vec4(a_Color.rgb * diffuse, a_Color.a);
```

**Comment by jdduke** [comment]
Hi @khrismuc, thanks for the feedback, we'll be sure to get this cleaned for the next release.

**Comment by jdduke** [comment]
This should be resolved with the latest 1.10.0 release, thanks for the feedback.
